### ISSUE TITLE
Can't access to other devices.

### ISSUE THREAD

**Issue by zl939144892** [zerotier/ZeroTierOne#1766]
- What you expect to be happening.
  Can access to other devices.
- What is actually happening?
  Can't access to other devices, but other device can access this device.
- Any steps to reproduce the error.
  Everytime. But only happened on my Mac.
- Any relevant console output or screenshots.
  Zerotier Network manage page:
  <img width="972" alt="image" src="https://user-images.githubusercontent.com/18364560/197972573-5e9bde9c-2037-47e0-81eb-ce587a51646e.png">

  MAC:
  <img width="570" alt="image" src="https://user-images.githubusercontent.com/18364560/197972685-e9227d33-e324-4bcc-945f-ea461f8da166.png">
  <img width="530" alt="image" src="https://user-images.githubusercontent.com/18364560/197973398-ab14d300-46fb-4d73-aaa3-6df9e4e97d50.png">

  Another device:
  <img width="530" alt="image" src="https://user-images.githubusercontent.com/18364560/197973628-591e06b6-84e6-4e6b-babc-34e9aad82ab2.png">

- What operating system and ZeroTier version. Please try the latest ZeroTier release. 
  1.10.1

**Comment by zl939144892** [comment]
Here is the dump:
[zerotier_dump.txt](https://github.com/zerotier/ZeroTierOne/files/9867561/zerotier_dump.txt)

**Comment by laduke** [comment]
Hello,
If ping works from one node, traffic _is_ working in both directions. The other node is likely blocking icmp in it's OS firewall.

**Comment by zl939144892** [comment]
Hi @laduke, thanks for your response, but the firewall was already disabled on my mac.
<img width="504" alt="image" src="https://user-images.githubusercontent.com/18364560/198221873-4217ab49-2cf1-43f1-aef1-ab972f2c4c2b.png">

**Comment by zl939144892** [comment]
@laduke Any idea?

**Comment by zl939144892** [comment]
@laduke 
`10.147.17.199` is the third device, as you see: the mac ping timeout, the other device ping success.
<img width="1589" alt="image" src="https://user-images.githubusercontent.com/18364560/199632037-2330280e-ee4d-4202-8763-2ce6e4b11c6e.png">

**Comment by laduke** [comment]
Maybe they are both behind a "hard" NAT. You can check `zerotier-cli peers` to see if they are connecting to each other. 
https://docs.zerotier.com/zerotier/troubleshooting#router-configuration-tips

**Comment by glimberg** [comment]
So to summarize:

1. You *can* ping from the cloud instance to the macbook air.
2. You *can't* ping from the Macbook to the Cloud instance

This means that ZeroTier is indeed working as pings require both a packet going out and coming back.   As @laduke said above, you likely have a firewall blocking pings.  If it's your cloud server that is not responding, it likely has firewall rules on it blocking the pings.

**Comment by zl939144892** [comment]
@glimberg  
Yes, your summary was right!
But in the Mac system setting, the firewall was already disabled:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/18364560/200719843-750ad166-3434-49f4-89ed-3f5591f27975.png">
Is it possible another reason, like another app blocking ping action?
Have any commands to diagnose the zerotire network for the Mac?

**Comment by glimberg** [comment]
As I stated above, it's not the mac firewall that is the likely problem.  It's the other end.  The mac is getting pings & replying showing ZeroTier is indeed working. It's the other end that is blocking pings.

**Comment by zl939144892** [comment]
The situation like this:
<img width="805" alt="image" src="https://user-images.githubusercontent.com/18364560/200789758-94ed40d7-4dff-4d7a-92f8-b7985b8fd5ae.png">

**Comment by glimberg** [comment]
Well I can't tell you why the 2 other devices can ping each other over ZeroTier and the Macbook cannot ping either of them.  But the fact that both of the devices can ping the mac over ZeroTier means that ZeroTier is working as packets need to flow in both directions for a ping to work.  Whatever the issue is, ZeroTier is not the problem.

**Comment by zl939144892** [comment]
That's too depressing.

**Comment by joseph-henry** [comment]
I have to agree with @glimberg that this does not appear to be a ZeroTier problem. A successful ping from inside the cloud to the mac (and back!) does mean it works in both directions. Seems like the cloud service is blocking some select traffic somewhere.

What cloud service are you using?

**Comment by cloverzyy** [comment]
这是来自QQ邮箱的假期自动回复邮件。来信已收到，辛苦各位啦！
